src/main/java/com/work/management/web/rest/controller/EmployeeController.java

```diff
@@ -41,4 +41,12 @@
     return new ResponseEntity<>(
         EmployeeAssembler.convert(employeeService.getEmployeeByUserName(userName)), HttpStatus.OK);
   }
+
+  @RequestMapping(method = RequestMethod.PUT, value = "/update", produces = "application/json; charset=UTF-8")
+  @ApiOperation(value = "Update existing employee entity", response = EmployeeResource.class)
+  ResponseEntity<EmployeeResource> updateEmployeeEntity(
+      @Valid @RequestBody @ApiParam(value = "Update employee entity", required = true) EmployeeResource employeeResource) {
+    employeeService.updateEmployeeEntity(EmployeeAssembler.convert(employeeResource));
```

Comment:
If you are returning updated employee value from employeeService.updateEmployee ..then convert to resourve and send that value in the Response...otherwise what's the point of returning it from that function.